buddies-server/routes/auth.js

```diff
@@ -33,37 +32,78 @@ router.post(
 
       // save user
       const user = await newUser.save();
-      res.status(200).json(user);
+
+      const token = generateToken({
+        id: user._id,
```

Comment:
`_id` is now being used in the backend but the frontend is still using `id`.

Reply:
> `_id` is now being used in the backend but the frontend is still using `id`.

Yes. Actually I didn't really like _id as an attribute name although I knew it was MongoDB's default for id, that's why I changed it to id on the frontend. But I guess consistency is more important. What do you think?

Reply:
Nevermind. Pushed a commit to resolve this